### ISSUE TITLE
Debian 10 dependency: libtinfo5

### ISSUE THREAD

**Issue by oylenshpeegul** [2mol/pboy#34]
I just tried [the 1.3 release](https://github.com/2mol/pboy/releases/tag/1.3) on my Debian 10 box, but it failed at first. Apparently, it has a newer libtinfo.

```
$ pboy
pboy: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory

$ dpkg -S libtinfo.so
libtinfo6:amd64: /lib/x86_64-linux-gnu/libtinfo.so.6.1
libtinfo6:amd64: /lib/x86_64-linux-gnu/libtinfo.so.6
libncurses-dev:amd64: /usr/lib/x86_64-linux-gnu/libtinfo.so
```
I tried tricking it by linking the older name to the newer library, but it didn't like that either.

```
$ cd /lib/x86_64-linux-gnu
$ sudo ln -s libtinfo.so.6 libtinfo.so.5
$ pboy
pboy: /lib/x86_64-linux-gnu/libtinfo.so.5: version `NCURSES_TINFO_5.0.19991023' not found (required by pboy)
```
Turns out there's a [libtinfo5 package](https://packages.debian.org/buster/libtinfo5) that can be installed alongside the existing one without a problem.

```
$ sudo apt install libtinfo5
```
After that, `pboy` works fine!

I guess adding a dependency on libtinfo5 would be the kind of thing you put in the .deb when you make it. Or should it be conditionally compiled against libtinfo.so.6 instead?

**Comment by 2mol** [comment]
Hi, thank you for reporting this and especially for going further and looking into the details! I'll have a look where this dependency even comes from.

In the meantime, feel free to add a little mention of this dependency to the Readme (I'm on mobile right now), and I'll happily merge it.

**Comment by 2mol** [comment]
fixed by #46
